app/router.js

```diff
@@ -20,9 +20,15 @@ const Router = EmberRouter.extend(RouterScroll, {
 /* eslint array-callback-return: 0 */
 Router.map(function () {
   this.route('article-preview');
-
+  this.route('closed-wiki', {
+    path: '/__closed_wiki__this_is_never_visible_to_user',
```

Comment:
nice path ;)

Reply:
Got the idea from https://github.com/emberjs/ember.js/blob/9830f066ae2ef4fad4bfa63f5cc43997d8ac466b/packages/%40ember/-internals/routing/lib/system/dsl.ts#L46 ;)